.github/workflows/ci-emulator.yaml

```diff
@@ -0,0 +1,42 @@
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+name: Integration tests against emulator
+jobs:
+  units:
+    runs-on: ubuntu-latest
+
+    services:
+      emulator:
+        image: gcr.io/cloud-spanner-emulator/emulator:latest
+        ports:
+          - 9010:9010
+          - 9020:9020
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: stCarolas/setup-maven@v4
+      with:
+        maven-version: 3.8.1
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - run: java -version
+    - run: gcloud config set auth/disable_credentials true
+    - run: gcloud config set project test-project
+    - run: gcloud config set api_endpoint_overrides/spanner http://localhost:9020/
+    - run: gcloud spanner instances create test-instance --config=emulator-config --nodes=1 --description="TestInstance"
+    - run: mvn clean compile
```

Comment:
Do we need this? Looks like `compile` runs in the next step anyway. Do you expect `clean` to do anything on first run?